### PR TITLE
chore: polish code of web console backend

### PR DESCRIPTION
**Pull request by tyyzqmf**
----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

## Summary

(describe what this merge request does)

## Implementation highlights

(describe how the merge request does for feature changes, share the RFC link if it has)

## Test checklist

- [ ] add new test cases
- [x] all code changes are covered by unit tests
- [ ] end-to-end tests
  - [ ] deploy control plane with CloudFront + S3 + API gateway
  - [ ] deploy control plane within VPC
  - [ ] deploy ingestion server
    - [ ] with MSK sink
    - [ ] with KDS sink
    - [ ] with S3 sink
  - [ ] deploy data processing
  - [ ] deploy data modeling
    - [ ] new Redshift Serverless
    - [ ] provisioned Redshift
    - [ ] Athena
  - [ ] deploy with reporting

## Is it a breaking change

- [ ] add parameters without default value in stack
- [ ] introduce new service permission in stack
- [ ] introduce new top level stack module

### CODE REVIEW

File: src/control-plane/backend/lambda/api/store/dynamodb/dynamodb-store.ts [hunk 1]
```diff
@@ -48,7 +48,7 @@ export class DynamoDbStore implements ClickStreamStore {
         region: project.region,
         environment: project.environment,
         pipelineId: '',
-        status: 'ACTIVED',
+        status: {},
```
Comment (zxkane) [inline src/control-plane/backend/lambda/api/store/dynamodb/dynamodb-store.ts:51]: Will it be a change to break existing data?

Comment (tyyzqmf) [reply src/control-plane/backend/lambda/api/store/dynamodb/dynamodb-store.ts:51]: This is an initial value that does not affect existing data.

Comment (zxkane) [reply src/control-plane/backend/lambda/api/store/dynamodb/dynamodb-store.ts:51]: How about using the `status` value with different types(`string` or `object`) when getting the project detail?

Comment (tyyzqmf) [reply src/control-plane/backend/lambda/api/store/dynamodb/dynamodb-store.ts:51]: The **pipeline.status** should be `PipelineStatus` object, so string is an error initial value.

Comment (tyyzqmf) [reply src/control-plane/backend/lambda/api/store/dynamodb/dynamodb-store.ts]: My mistake, I mistook it for **pipeline.status**, and **project.status**  is indeed a `string` type. And reset this attribute is `ACTIVATED`.

**Review by zxkane — APPROVED** [review]

**Pull request merged**